README.md

```diff
@@ -20,15 +30,13 @@ By default, there exists no `.env.custom` file. In this case, you can manually a
 
 Please keep in mind to check the `.env` file for changes, when you perform an upgrade of Sentry, so that you can adjust your `.env.custom` accordingly, if required.
 
-### Installation
+### Enhance Sentry image
 
-To get started with all the defaults, simply clone the repo and run `./install.sh` in your local check-out. Sentry uses Python 3 by default since December 4th, 2020 and Sentry 21.1.0 is the last version to support Python 2.
+To install plugins and their dependencies or make other modifications to the Sentry base image,
+copy `sentry/enhance-image.example.sh` to `sentry/enhance-image.sh` and add necessary steps there.
+For example, you can use `apt-get` to install dependencies and use `pip` to install plugins.
 
-During the install, a prompt will ask if you want to create a user account. If you require that the install not be blocked by the prompt, run `./install.sh --skip-user-prompt`.
-
-Thinking of not managing this yourself? Check out the [SaaS migration docs](https://docs.sentry.io/product/sentry-basics/migration/) or [contact us](https://sentry.io/from/self-hosted) for help.
-
-Please visit [our documentation](https://develop.sentry.dev/self-hosted/) for everything else.
+After making modifications to `sentry/enhance-image.sh`, run `./install.sh` again to apply them.
```

Comment:
Glad to see docs for this, I'm torn between README and develop.sentry.dev/self-hosted. I think as with the .env customization we should dupe over there for now and we can dedupe in some glorious future where we fold all README content into dev docs.

Reply:
Duping in https://github.com/getsentry/develop/pull/590.